devtools/create-gossipstore.c

```diff
@@ -4,24 +4,44 @@
 #include <ccan/read_write_all/read_write_all.h>
 #include <common/type_to_string.h>
 #include <common/utils.h>
+#include <devtools/create-gossipstore.h>
 #include <fcntl.h>
 #include <gossipd/gen_gossip_store.h>
 #include <gossipd/gossip_store.h>
 #include <inttypes.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
 #include <wire/gen_peer_wire.h>
 
+
+
+struct scidsat * load_scid_file(FILE * scidfd) 
+{
+        int n;
+        fscanf(scidfd, "%d\n", &n);	
+	char title[16];
+        fscanf(scidfd, "%s\n", title);	
+	struct scidsat * scids = calloc(n, sizeof(scidsat));
+	int i = 0;
+        while(fscanf(scidfd, "%s ,%ld\n", scids[i].scid, &scids[i].sat.satoshis) == 2 ) {
+			i++;
+	}
+	return scids;
+}
+
 int main(int argc, char *argv[])
 {
 	u8 version;
 	beint16_t be_inlen;
 	struct amount_sat sat;
 	bool verbose = false;
-	char *infile = NULL, *outfile = NULL;
+	char *infile = NULL, *outfile = NULL, *scidfile = NULL, *csat = NULL;
 	int infd, outfd;
+       	FILE * scidfd;
```

Comment:
Trivial: space after `*`.
More importantly, `fd` is used for file descriptors, and this isn't an fd but a FILE *.  `scidf` would work.